### PR TITLE
CEDS_602 refactored error handling for frontend pages

### PR DESCRIPTION
**Pull request by raghu1978**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -27,7 +27,7 @@ import features.FeatureStatus.FeatureStatus
 import uk.gov.hmrc.play.config.{AppName, ServicesConfig}
 
 @Singleton
-class AppConfig @Inject() (override val runModeConfiguration: Configuration, environment: Environment)
+class AppConfig @Inject() (override val runModeConfiguration: Configuration, val environment: Environment)
```
Comment (PatrykRudnicki) [inline app/config/AppConfig.scala]: Why you need `val` here?

File: app/controllers/SimpleDeclarationController.scala [hunk 2]
```diff
@@ -56,8 +56,9 @@ class SimpleDeclarationController @Inject()(appConfig: AppConfig,
   val form = Form(mapping(
       "ducr" -> nonEmptyText.verifying(pattern(correctDucrFormat.r, error = "error.ducr")),
       "isConsolidateDucrToWiderShipment" -> boolean,
-      "mucr" -> mandatoryIfTrue("isConsolidateDucrToWiderShipment",
+     "mucr" -> mandatoryIfTrue("isConsolidateDucrToWiderShipment",
         nonEmptyText.verifying(pattern("""^[A-Za-z0-9 \-,.&'\/]{1,65}$""".r, error = "error.ducr"))),
+//    "mucr" -> optional(text),
```
Comment (PatrykRudnicki) [inline app/controllers/SimpleDeclarationController.scala]: Please remove comment

File: app/controllers/SimpleDeclarationController.scala [hunk 3]
```diff
@@ -25,7 +25,7 @@ import javax.inject.Inject
 import models.CustomsDeclarationsResponse
 import play.api.Logger
 import play.api.data.Form
-import play.api.data.Forms.{boolean, mapping, nonEmptyText, text}
+import play.api.data.Forms.{boolean, mapping, nonEmptyText, text, optional,ignored}
```
Comment (PatrykRudnicki) [inline app/controllers/SimpleDeclarationController.scala]: You don't need these two imports

File: app/config/AppConfig.scala (hunk 1 above)
Comment (raghu1978) [reply app/config/AppConfig.scala]: this is to inject the environment

File: app/controllers/SimpleDeclarationController.scala (hunk 3 above)
Comment (raghu1978) [reply app/controllers/SimpleDeclarationController.scala]: agree , removed.

File: app/controllers/SimpleDeclarationController.scala (hunk 2 above)
Comment (raghu1978) [reply app/controllers/SimpleDeclarationController.scala]: removed.

**Pull request merged**